packages/experiments/src/components/Button/Button.test.tsx

```diff
@@ -99,4 +100,26 @@ describe('Button', () => {
     const tree = component.toJSON();
     expect(tree).toMatchSnapshot;
   });
+
+  it('focuses correctly when focus is triggered via IButton interface', () => {
+    const wrapper = mount(
+      <div>
+        <Button content="Button 1" />
+        <Button content="Button 2" />
+        <Button content="Button 3" />
+      </div>
+    );
+
+    const buttons = wrapper.getDOMNode().querySelectorAll('button.ms-Button') as NodeListOf<HTMLButtonElement>;
```

Comment:
I am confused. This isn't testing `IButton` interface at all. You should be adding a `componentRef` to these buttons and calling `focus` on `ref.current` resolved from that.

What's happening here is that you're selecting DOM elements and focusing them. You're not actually testing any logic we own.

Reply:
I see now, I don't know what I was thinking yesterday, I'll change it so it correctly reflects what I'm trying to do.